neo4j/db/errors.go

```diff
@@ -23,12 +23,34 @@ import (
 	"strings"
 )
 
+type ErrorClassification string
+
+const (
+	ClientError    ErrorClassification = "CLIENT_ERROR"
+	DatabaseError  ErrorClassification = "DATABASE_ERROR"
+	TransientError ErrorClassification = "TRANSIENT_ERROR"
+	UnknownError   ErrorClassification = "UNKNOWN"
+)
+
 // Neo4jError is created when the database server failed to fulfill request.
+// TODO 6.0: rename to GqlError
```

Comment:
As discussed on the call, reconsider if this (and the other to dos for 6.0)  should remain. Maybe only some, maybe all, maybe none.

If you decide to keep some/all, please be aware that these comments show up in the API docs. :grimacing: not so nice.

Reply:
I moved the TODOs regarding future renaming above the structure so as not to interfere with API docs.